Missing lodash dependency for admin js package

### Steps to reproduce

Precompile the assets with `RAILS_ENV=production bin/rake assets:precompile`

### Expected behavior
No error when precompiling the assets.

### Actual behavior

I get the following error:

```
COMPILATION FAILED:
EXIT STATUS: pid 19245 exit 2
OUTPUTS:
[webpack-cli] ModuleNotFoundError: Module not found: Error: Can't resolve 'lodash/debounce' in '/home/antonio/Projects/SmartMirror/Server/node_modules/@alchemy_cms/admin/package/src'
    at /home/antonio/Projects/SmartMirror/Server/node_modules/webpack/lib/Compilation.js:2015:28
    at /home/antonio/Projects/SmartMirror/Server/node_modules/webpack/lib/NormalModuleFactory.js:798:13
    at eval (eval at create (/home/antonio/Projects/SmartMirror/Server/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /home/antonio/Projects/SmartMirror/Server/node_modules/webpack/lib/NormalModuleFactory.js:270:22
    at eval (eval at create (/home/antonio/Projects/SmartMirror/Server/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /home/antonio/Projects/SmartMirror/Server/node_modules/webpack/lib/NormalModuleFactory.js:434:22
    at /home/antonio/Projects/SmartMirror/Server/node_modules/webpack/lib/NormalModuleFactory.js:116:11
    at /home/antonio/Projects/SmartMirror/Server/node_modules/webpack/lib/NormalModuleFactory.js:670:25
    at /home/antonio/Projects/SmartMirror/Server/node_modules/webpack/lib/NormalModuleFactory.js:855:8
    at /home/antonio/Projects/SmartMirror/Server/node_modules/webpack/lib/NormalModuleFactory.js:975:5
resolve 'lodash/debounce' in '/home/antonio/Projects/SmartMirror/Server/node_modules/@alchemy_cms/admin/package/src'
  Parsed request is a module
  using description file: /home/antonio/Projects/SmartMirror/Server/node_modules/@alchemy_cms/admin/package.json (relative path: ./package/src)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      looking for modules in /home/antonio/Projects/SmartMirror/Server/app/frontend
        /home/antonio/Projects/SmartMirror/Server/app/frontend/lodash doesn't exist
      /home/antonio/Projects/SmartMirror/Server/node_modules/@alchemy_cms/admin/package/src/node_modules doesn't exist or is not a directory
      /home/antonio/Projects/SmartMirror/Server/node_modules/@alchemy_cms/admin/package/node_modules doesn't exist or is not a directory
      /home/antonio/Projects/SmartMirror/Server/node_modules/@alchemy_cms/admin/node_modules doesn't exist or is not a directory
      /home/antonio/Projects/SmartMirror/Server/node_modules/@alchemy_cms/node_modules doesn't exist or is not a directory
      /home/antonio/Projects/SmartMirror/Server/node_modules/node_modules doesn't exist or is not a directory
      looking for modules in /home/antonio/Projects/SmartMirror/Server/node_modules
        /home/antonio/Projects/SmartMirror/Server/node_modules/lodash doesn't exist
      /home/antonio/Projects/SmartMirror/node_modules doesn't exist or is not a directory
      /home/antonio/Projects/node_modules doesn't exist or is not a directory
      /home/antonio/node_modules doesn't exist or is not a directory
      /home/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
```

### System configuration

- Alchemy Version: 6.0.0.pre.rc6
- Rails Version: 6.1.5

I see a dependency from `lodash-es` instead of `lodash`, maybe is it enough to require a different module [here](https://github.com/AlchemyCMS/alchemy_cms/blob/main/package/src/picture_editors.js#L1) or adding the lodash dependency [here](https://github.com/AlchemyCMS/alchemy_cms/blob/main/package.json#L28)? I can create a PR once agree on the direction to fix this issue.
In development it's fine because I have other packages that depends on `lodash`, so the package is there anyway in the `node_modules` directory.

--- a/package/src/picture_editors.js
+++ b/package/src/picture_editors.js
@@ -1,5 +1,5 @@
-import debounce from "lodash/debounce"
-import max from "lodash/max"
+import debounce from "lodash-es/debounce"
+import max from "lodash-es/max"
 import { get } from "./utils/ajax"
 import ImageLoader from "./image_loader"
 

